Yijun dev2022

ymy5178 opened the pull request:
please take a short time to review. Thank you so much!

ymy5178 committed:
first edition

random success 0
feedback

ymy5178 committed:
final edit

ymy5178 committed:
references

ymy5178 committed:
cite & screenshot

ymy5178 committed:
update restart button

NurulSyafiqah-pixel reviewed (commented):
The NA appears after multiple 'restart' click. But less than before the first review.

Xinyue-Tang reviewed (dismissed):
It is working on my laptop!

ymy5178 committed:
reset button

ymy5178 committed:
solve the warning produced by bernoulli

ymy5178 commented:
@NurulSyafiqah-pixel 
Can you help me to review again?
I checked all the three scenarios in my laptop, and i tried 15 'restart' click in each one and they all work.
Thank you so much!

ymy5178 committed:
alt and na.rm = TRUE

add alt in graphs. add na.rm = TRUE in geom_step()

neilhatfield committed:
Create screenshot_temp.png

npechulis committed:
small fixes

-grammar fixes
-aesthetics error fixed
-clarified instructions

npechulis committed:
more small changes

npechulis committed:
fix hint pmf graph

adjusted axis title sizes, added plot titles, and fit the data to the graph better

npechulis committed:
add background info and adjust instructions

npechulis committed:
minor fixes

npechulis committed:
adjusted prereq info mostly

npechulis committed:
fixed aria captions

npechulis committed:
formatting fixes

npechulis committed:
mostly explore page fixes

npechulis committed:
game functionality fixes

npechulis committed:
fixes from beta testing

npechulis committed:
prereq fixes

neilhatfield committed:
Streamlining Code

Changes made to try streamlining code

neilhatfield committed:
Delete test.csv

Remove extra temp file

neilhatfield committed:
Move Files

Move a couple of background files to the docs folder

neilhatfield committed:
Update app.R

Add check for null answer input

neilhatfield committed:
Final Clean up

final bits of clean up

neilhatfield committed:
Merge pull request #3 from EducationShinyAppTeam/dev_Neil2024

Dev neil2024

neilhatfield committed:
Merge pull request #4 from EducationShinyAppTeam/devNathan2024

Dev nathan2024
